src/get-authorization-url.ts

```diff
@@ -1,12 +1,16 @@
 import { workos } from './workos.js';
 import { WORKOS_CLIENT_ID, WORKOS_REDIRECT_URI } from './env-variables.js';
+import { GetAuthURLOptions } from './interfaces.js';
+
+async function getAuthorizationUrl(options: GetAuthURLOptions) {
+  const { returnPathname, screenHint } = options;
 
-async function getAuthorizationUrl(returnPathname?: string) {
   return workos.userManagement.getAuthorizationUrl({
     provider: 'authkit',
     clientId: WORKOS_CLIENT_ID,
     redirectUri: WORKOS_REDIRECT_URI,
     state: returnPathname ? btoa(JSON.stringify({ returnPathname })) : undefined,
+    screenHint: screenHint || undefined,
```

Comment:
I think this could be this right?

```suggestion
    screenHint,
```